src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -662,6 +689,20 @@ private function rowToArray(Row $row, $locale, $webspaceKey, $fields, $onlyPubli
             return false;
         }
 
+        if ($document instanceof SecurityBehavior && $permission) {
+            $permissionKey = \array_search($permission, $this->permissions);
+            $permissions = $this->accessControlManager->getUserPermissionByArray(
```

Comment:
Could be null if website has no security enabled.

Reply:
Same as above